docs/community_plugins.md

```diff
@@ -0,0 +1,31 @@
+# Community Plugins
+
+Share your custom plugin with others by adding it to the list using templete below! Check out the [contribution guide](https://github.com/dsrvlabs/vatz/blob/main/docs/contributing.md) and PR to update. (PR will be reviewed by DSRV)
+You could remove it via PR, but only if the person who added it to the list requests to be removed from the list.
+
+> ### 1. Plugin Name
+> 1. Summary: Describe what is your plugin for
+> 2. Github: Github address of your plugin repository 
+> 3. Version: https://pkg.go.dev link for latest version.
+> 4. Usage: Add link to a guide on how to intall and use your plugin.
+---
+## List
+### 1. vatz-plugin-sysutil (example)
```

Comment:
@qkrybin 
Thanks for including this on documents
Can you update plugin name that is listed on 
- https://github.com/dsrvlabs/vatz/pull/457
   - vatz-plugin-sysutil
      - vatz_cpu_monitor
      - vatz_mem_monitor
      - vatz_disk_monitor
      - vatz_net_monitor
   - vatz-cosmos-hub
      - vatz_block_sync
      - vatz_node_is_alived
      - vatz_peer_count
      - vatz_active_status
      - vatz_gov_alarm

Reply:
updated